final/design.tex

```diff
@@ -1,6 +1,49 @@
 \chapter{Design}
 \label{cha:design}
 
+As detailed in the previous chapters, the Spoofax language workbench already
```

Comment:
Currently all mentions of "Spoofax Language Workbench" use a capitalized L and W. I have no preference, but we should be consistent. Do we change this one occurrence here to uppercase, or do we change the other occurences to lowercase?